content/_code-samples/non-fungible-token/py/create-sell-offer-nft.py

```diff
@@ -0,0 +1,86 @@
+from xrpl.models.transactions.nftoken_create_offer import NFTokenCreateOffer, NFTokenCreateOfferFlag
+from xrpl.transaction import safe_sign_and_autofill_transaction, send_reliable_submission
+from xrpl.models.requests import AccountNFTs
+from xrpl.clients import JsonRpcClient
+from xrpl.models import NFTSellOffers
+from xrpl.wallet import Wallet
+
+# Put up a NFT sell offer on the XRPL via NFTokenCreateOffer transaction with flag TF_SELL_NFTOKEN
+# https://xrpl.org/nftokencreateoffer.html#nftokencreateoffer
+
+# For this code snippet to work, you need import an Account Seed with an already issued NFT
+# You could use 'mint_nft.py' on the same folder to import an account
+
+seed = ""
+
+# Connect to a testnet node
+print("Connecting to Testnet...")
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Initialize wallet from seed
+issuer_wallet = Wallet(seed=seed, sequence=0)
```

Comment:
Note: Passing an empty string into a Wallet gives a very opaque error, I changed `authorize-minter.py` to follow different steps to avoid the issue, but feel free to decide on a different/better way of handling it :) 

```
Traceback (most recent call last):
  File "/Users/jacksonmills/Documents/bounty_submissions/xrpl-dev-portal/content/_code-samples/non-fungible-token/py/create-sell-offer-nft.py", line 22, in <module>
    issuer_wallet = Wallet(seed=seed, sequence=0)
  File "/Users/jacksonmills/Library/Caches/pypoetry/virtualenvs/xrpl-validator-dashboard-mm7G7WMB-py3.9/lib/python3.9/site-packages/xrpl/wallet/main.py", line 41, in __init__
    pk, sk = derive_keypair(self.seed, algorithm=algorithm)
  File "/Users/jacksonmills/Library/Caches/pypoetry/virtualenvs/xrpl-validator-dashboard-mm7G7WMB-py3.9/lib/python3.9/site-packages/xrpl/core/keypairs/main.py", line 69, in derive_keypair
    decoded_seed, algorithm = addresscodec.decode_seed(seed, algorithm)
  File "/Users/jacksonmills/Library/Caches/pypoetry/virtualenvs/xrpl-validator-dashboard-mm7G7WMB-py3.9/lib/python3.9/site-packages/xrpl/core/addresscodec/codec.py", line 123, in decode_seed
    decoded_result = _decode(seed, bytes(prefix))
  File "/Users/jacksonmills/Library/Caches/pypoetry/virtualenvs/xrpl-validator-dashboard-mm7G7WMB-py3.9/lib/python3.9/site-packages/xrpl/core/addresscodec/codec.py", line 61, in _decode
    decoded = base58.b58decode_check(b58_string, alphabet=XRPL_ALPHABET)
  File "/Users/jacksonmills/Library/Caches/pypoetry/virtualenvs/xrpl-validator-dashboard-mm7G7WMB-py3.9/lib/python3.9/site-packages/base58/__init__.py", line 157, in b58decode_check
    raise ValueError("Invalid checksum")
ValueError: Invalid checksum
```

Reply:
Done, thank you!